skins/oasis/modules/templates/Oasis_Index.php

```diff
@@ -74,6 +74,7 @@
 <?= $dynamicYield ?>
 <?= $ivw2 ?>
 <?= $rubiconRtp ?>
+<gcse:search></gcse:search>
```

Comment:
`wgGoogleSearchTest` is set via Optimizely so we can't affect that back end base on that variable. That said, it would be nice if it was included via template the same way the code snippets above it are.